.travis.yml

```diff
@@ -1,5 +1,63 @@
-language: node_js
-node_js:
-  - "node"
+sudo: false
+
+language:
+  - php
+  - node_js
```

Comment:
Only a single language is supported by Travis CI, keep `- php` and remove `- node_js`